core/server/data/migrations/versions/3.18/001-add-members_stripe_connect-auth-permissions.js

```diff
@@ -0,0 +1,56 @@
+const ObjectId = require('bson-objectid');
+const {logging} = require('../../../../lib/common');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+    async up(options) {
+        const connection = options.transacting;
+
+        const existingIdentityPermission = await connection('permissions').where({
+            action_type: 'auth',
+            object_type: 'members_stripe_connect'
+        }).first();
+
+        if (existingIdentityPermission) {
+            logging.warn('Permission for auth:members_stripe_connect already added');
+            return;
+        }
+
+        logging.info('Adding permission for auth:members_stripe_connect');
+
+        const date = connection.raw('CURRENT_TIMESTAMP');
+
+        await connection('permissions').insert({
+            id: ObjectId.generate(),
+            name: 'Auth Stripe Connect for Members',
+            action_type: 'auth',
+            object_type: 'members_stripe_connect',
+            created_at: date,
+            created_by: 1,
```

Comment:
Think using hardcoded `1` here is not the safest approach for couple reasons: 1) one can transfer "owner" role to other user 2) one can remove the user after the transfer which would make this piece break (no user with `id:1`).
Noticed this logic is used in [another migration](https://github.com/TryGhost/Ghost/blob/935acfdb88824f24e04a13b8a0ca647066ba2a36/core/server/data/migrations/versions/3.12/01-add-identity-permission.js#L31) as well. Suggest we fix this. Same goes for `updated_by` column.

Reply:
This is what would be used if set by the model layer https://github.com/TryGhost/Ghost/blob/d2cf7111ec3a77a0f6454783a43ea79ddb6da3ae/core/server/models/base/index.js#L661

Fixing this is way outside the scope of this issue IMO

Reply:
@allouis there is very little code that relies on the "internalUser" and `1` as hardcoded value. There was a push towards eliminating use of it in this [issue](https://github.com/TryGhost/Ghost/issues/9299).  I think adding these hard-coded values in migrations adds to a problem which hasn't been fully eliminated. Would rather go with additional query to figure out what is the current "owner id". Alternatively, would add this piece of code to be tracked in the issue above, so once it's on the table again we know where to clean up. 

One thing to bear in mind, having these `1`s in migrations "cements" them here for long time as we can't retroactively fix them.

Reply:
This issue is also on the [roadmap](https://github.com/TryGhost/Team/issues/221) for Ghost 4.0 under "Review v0.1 related "dirty hotfix" in contextUser method"

Reply:
>  I think adding these hard-coded values in migrations adds to a problem which hasn't been fully eliminated.

From what I can tell, literally every migration we currently have uses the id of `1` for the {created,updated}_by fields.

Until we have a plan of how it should be handled I don't think we should start changing things now and deviating from the norm - what do you think?

Reply:
Yup, agreed. Forgot about `context:internal` using the `internalUser` id in the end of the day :disappointed: Let's roll with the `1` :+1: